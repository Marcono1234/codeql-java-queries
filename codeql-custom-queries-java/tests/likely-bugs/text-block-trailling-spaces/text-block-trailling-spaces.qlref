./likely-bugs/text-block-trailling-spaces.ql
