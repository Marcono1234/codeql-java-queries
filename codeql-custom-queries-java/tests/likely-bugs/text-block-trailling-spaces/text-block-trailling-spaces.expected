| Test.java:3:9:6:11 | """\n        spaces   \n        second line   \n        """ | Has trailing whitespaces at text block line 0 (0-based) |
| Test.java:3:9:6:11 | """\n        spaces   \n        second line   \n        """ | Has trailing whitespaces at text block line 1 (0-based) |
| Test.java:7:9:9:11 | """\n        tabs\t\t\n        """ | Has trailing whitespaces at text block line 0 (0-based) |
| Test.java:10:9:11:24 | """\n        same line    """ | Has trailing whitespaces at text block line 0 (0-based) |
